app/src/full/java/io/homeassistant/companion/android/settings/wear/SettingsWearViewModel.kt

```diff
@@ -280,6 +282,7 @@ class SettingsWearViewModel @Inject constructor(
         favoriteEntityIdList.forEach { entityId ->
             favoriteEntityIds.add(entityId)
         }
+        // TODO: forward and backward compatibility?
         setTemplateContent(data.getString(WearDataMessages.CONFIG_TEMPLATE_TILE, ""))
         templateTileRefreshInterval.value = data.getInt(WearDataMessages.CONFIG_TEMPLATE_TILE_REFRESH_INTERVAL, 0)
```

Comment:
Please help me out here, what do we need to support?
- Old mobile app + new Wear OS app?
- New mobile app + old Wear OS app?
- Or can we always assume that both apps are of the same version?

Reply:
We generally assume the versions match and don't try to keep compatibility with older app combinations.

Reply:
play store is updated with both required versions at the same time too so its a safe assumption there :)

Reply:
Ah, great :+1: